documents/templates/commcomm_to_tsc.md

```diff
@@ -0,0 +1,7 @@
+```
```

Comment:
>Using a template encourages fitting a square peg in a round hole.

Definitely agree on there being an issue with that - my main concern is that creating an issue in the CC and the TSC repos could honestly be a barrier to those interested in this process, but I may be mixing optimizing the experience of someone _new_ to the project versus someone who is already active and is aware of these processes.

Do you think this could be a possible barrier? If not, I'm +1 to completely removing these templates.